index.html

```diff
@@ -24,8 +24,9 @@ <h2><input type="text" class="nes-input" placeholder="what needs to be done" v-o
     </li>
   </ul>
 
-  <center><button type="button">Active</button>
-  <button type="button">Completed</button></center>
+  <center>
+  <button v-on:click="swapToCompleted">Completed</button>
+  <button v-on:click="swapToActive">Active</button>

```

Comment:
change `swapToActive` to `showActive`
change `swapToCompleted` to `showCompleted`